pmm-app/tests/verifyPMMSettingsPageElements_test.js

```diff
@@ -184,7 +184,7 @@ Scenario(
 );
 
 Scenario(
-  'PMM-T254 Verify validation for STT and Telemetry switches ',
+  'PMM-T254 Verify validation for STT and Telemetry switches',
```

Comment:
missing on this and many others

Reply:
They leverage the default severity )

Reply:
yes realized, I missed the default was being set